src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,147 @@
+import React, {Component, createRef} from 'react';
+import uniqId from 'uniqid';
+import {Button} from '../Button/Button';
+import './AddUserForm.scss';
+import ImgWarning from './../../assets/warning2.png'
+
+const CN = 'user-form';
+
+class AddUserForm extends Component {
+    constructor(props) {
+        super(props);
+        // this.userNameRef = createRef(); // аналогично записи в строке 19
+        // this.userLastNameRef = createRef();
+        this.state = {
+            currentError: '',
+            errorUserName: ' ',
```

Comment:
extra space should be deleted
`errorUserName: ' ',` -- without space inside quotes